hosting_docs/source/reference/1-commcare-cloud/1-installation.rst

```diff
@@ -117,12 +117,12 @@ If you get to this point, congratulations! ``commcare-cloud`` is installed.
 Manual Installation
 -------------------
 
-You will need python 3.6 installed to follow these instructions (for
-Ubuntu 18.04; other operating systems may differ):
+You will need python 3.10 installed to follow these instructions (for
+Ubuntu 22.04; other operating systems may differ):
```

Comment:
I'm not sure we want to reference 22.04 in public docs just yet, but I do agree with the 3.10 change.